tests/components/person/test_init.py

```diff
@@ -189,6 +189,43 @@ def storage_setup(hass, hass_storage, hass_admin_user):
     assert state.attributes.get(ATTR_USER_ID) == user_id
 
 
+async def test_ignore_unavailable_states(hass, hass_admin_user):
+    """Test set up person with two device trackers, one unavailable."""
+    user_id = hass_admin_user.id
+    config = {DOMAIN: {
+        'id': '1234', 'name': 'tracked person', 'user_id': user_id,
+        'device_trackers': [DEVICE_TRACKER, DEVICE_TRACKER_2]}}
+    assert await async_setup_component(hass, DOMAIN, config)
+
+    state = hass.states.get('person.tracked_person')
+    assert state.state == STATE_UNKNOWN
+
+    hass.bus.async_fire(EVENT_HOMEASSISTANT_START)
+    await hass.async_block_till_done()
+    hass.states.async_set(DEVICE_TRACKER, 'home')
+    await hass.async_block_till_done()
+    hass.states.async_set(DEVICE_TRACKER, 'unavailable')
+    await hass.async_block_till_done()
+
+    # Unknown, as only 1 device tracker has a state, but we ignore that one
+    state = hass.states.get('person.tracked_person')
+    assert state.state == STATE_UNKNOWN
```

Comment:
Should this be state `'home'` since the device tracker had that state first, before becoming unknown state.

Reply:
The moment device tracker updates to unavailable, it's no longer home. So person should not rely on a no-longer available state in current implementation. I guess we can tweak in future.

Reply:
But we don't set the person state explicitly to unknown. We just ignore unknown or unavailable device tracker states when reacting to the device tracker change. I don't understand why this test works.

Reply:
Because I changed how the person entity determines state. It will fetch all new states for each entity and decides which one to take. So by overriding the only state that was there to one that is ignored, no state will be found.

Reply:
Yeah, I forgot that we do set state explicitly to `None` after check at line 380.